code/src/css/moviedetails.css

```diff
@@ -0,0 +1,68 @@
+.details-section {
+  display: flex;
+  flex-direction: column;
+  margin-left: 25px;
+}
+
+.poster-img {
+  height: 300px;
+  width: 200px;
+  border: 4px solid white;
+}
+
+.movie-title, .movie-rating, .movie-overview {
+  color: white;
+}
+
+.info-child {
+  max-width: 35ch;
+}
+
+.details-container {
+  margin-top: 35px;
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: center;
+  gap: 15px;
+}
+
+/* TABLET SIZE */
+@media screen and (min-width: 667px) {
+
+  .details-container {
+    margin: 20px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    gap: 55px;
+    align-items: center;
+    height: 100vh;
+  }
+  
+  .poster-img {
+    max-width: 140%;
+    width: 300px;
+    height: 90%;
```

Comment:
The poster text becomes a little blocked by the poster image in the details view right around the breakpoint for mobile screen and I think it's due to the settings on the poster image. If you remove the max-width 140% the problem goes away I think but I would play around a little in the inspect tool and you can see which exact values you think looks best :)